src/js/AppHeader.js

```diff
@@ -16,29 +16,14 @@ import uiController from './Controllers/UIController'
 import ScrollableMessage from './ScrollableMessage';
 
 import {ReactComponent as IconMenu} from '../img/icons/icon-menu.svg'
-import {ReactComponent as IconCart} from '../img/icons/icon-cart.svg'
-import {ReactComponent as PermissionsIcon} from '../img/static/0x49.svg'
 
-class AppStoreIcon extends React.Component {
+class MainMenuSettings extends React.Component {
     render() {
-        return (<div style={{ display: 'flex', flexDirection: 'row', alignItems: 'center' }}>
-                <Link to="/permissionapplist" style={{ marginRight: 10, marginLeft: 'auto' }}>
-                    <div className="app-icon">
-                        <div className="static-icon">
-                            <div className="svg-wrap">
-                                <PermissionsIcon/>
-                            </div>
-                        </div>
-                    </div>
-                </Link>
-                <Link to="/appstore" style={{ marginRight: 0 }}>
-                    <div className="app-icon">
-                        <div className="static-icon">
-                            <div className="svg-wrap">
-                                <IconCart/>
-                            </div>
-                        </div>
-                    </div>
+        return (<div>
+                <Link to="/settings">
+                    <span className="svg-wrap">
+                        <IconMenu/>
```

Comment:
I might suggest picking a different icon for this, I feel like it's a bit unclear as-is to just use the normal menu icon. Might be better to go with either a "gear" (settings) icon or a "home" icon (in which case, this screen would be added as an item in this menu)

Reply:
Switched to use the gear icon. Since this icon was being used by the permissions view, i switched the permissions view to use the lock static icon.